### PR TITLE
Add item callback

### PR DESCRIPTION
**Pull request by pietvr**

### CODE REVIEW

File: doc/configuration.rst [hunk 1]
```diff
@@ -332,6 +332,26 @@ In the example below the special item has ID *DOC-NOTIFICATION*.
...
+-------------------------------------------------
+Item callbacks, set optional attributes mandatory
+-------------------------------------------------
+
+When using the traceability plugin callback functionality on each item, a combination of 2 attributes
+`non_functional` and `functional` in an item can be made mandatory in the example below.
+
+.. code-block:: python
+
+    item = collection.get_item(name)
```
Comment (Letme) [inline doc/configuration.rst:347]: how about you add also function name and that this is to be put into Sphinx `conf.py`

Comment (Letme) [reply doc/configuration.rst:347]: and maybe we move this into section on line 285?

Comment (pietvr) [reply doc/configuration.rst]: https://github.com/melexis/sphinx-traceability-extension/pull/229/commits/929ae647677c6630d2e13c02ca7ee426895ecb74

Comment (pietvr) [reply doc/configuration.rst]: This doc file is about the config. No need to repeat.

**Pull request merged**